src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/PaymentOrderReversalsDto.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SwedbankPay.Sdk.PaymentInstruments;
+using SwedbankPay.Sdk.PaymentInstruments.Card;
+using System;
+using System.Collections.Generic;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    internal class PaymentOrderReversalsDto

```

Comment:
I think it's best that the DTO is named exactly like the core class it maps to and from, plus the `Dto` suffix.

```suggestion
    internal class ReversalListResponseDto
```

Reply:
I've done my best to so do, some times I do miss a renaming. Please point out any other inconsistencies you find!